packages/messaging/lib/index.d.ts

```diff
@@ -628,9 +694,9 @@ export namespace FirebaseMessagingTypes {
      *   });
      * ```
      *
-     * @param options Options to override senderId (iOS) and projectId (Android).
+     * @param options Options can be either `GetTokenOptions` or `NativeTokenOptions`
      */
-    getToken(options?: TokenOptions): Promise<string>;
+    getToken(options?: GetTokenOptions | NativeTokenOptions): Promise<string>;
```

Comment:
And this is the correct syntax to get the composite / all-members type, if I actually understand it now 

https://www.typescriptlang.org/docs/handbook/2/objects.html#intersection-types

```suggestion
    getToken(options?: GetTokenOptions & NativeTokenOptions): Promise<string>;
```